### PR TITLE
Switched the order of the company tabs

### PR DESCRIPTION
**Pull request by PaulWheatcroft**
## Description of change

Set the overview tab as the default view on the company page

## Test instructions

Click on a company and see that the tab displayed is the Overview

### Before

Screenshot 2023-03-28 at 10 27 40

### After

Screenshot 2023-03-28 at 10 28 07

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [x] Has the branch been rebased to main?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: test/functional/cypress/specs/companies/add-company-spec.js [hunk 1]
```diff
@@ -575,7 +575,7 @@ describe('Add company form', () => {
     it('should redirect to the new company activity', () => {
```
Comment (cgsunkel) [inline test/functional/cypress/specs/companies/add-company-spec.js:575]:
```suggestion
    it('should redirect to the new company overview', () => {
```

Comment (cgsunkel) [reply test/functional/cypress/specs/companies/add-company-spec.js:575]: This should be changed if the user isn't being redirected to the activity feed anymore

**Review by cgsunkel — APPROVED** [review]
LGTM, I only have one very minor comment but will pre-approve

**Review by simon-mt — APPROVED** [review]
Approved - agree with Chris' suggested change though.

**Pull request merged**